app/screens/ProductDetailsScreen/ProductDetailsScreen.tsx

```diff
@@ -47,19 +47,17 @@ export const ProductDetailsScreen: React.FC = ({ route, navigation }): JSX.Eleme
 
```

Comment:
PDP View is transparent

![Screenshot_1644242067](https://user-images.githubusercontent.com/58725560/152801326-b65c3e5f-02ae-46d6-9b6b-57450e7e857b.png)